core/capabilities/targets/write_target.go

```diff
@@ -0,0 +1,423 @@
+package targets
```

Comment:
This is hard to review as ad-hoc changes, especially if not intimately familiar with what was there before. Can you try to express this change as a `git revert` of the specific commit we're trying to revert (https://github.com/smartcontractkit/chainlink/commit/654c21ed89756dcbe9a9496d0ce8a55601901c5b) + any changes on top in a separate commit. That would be a lot easier to verify.

Reply:
I separated them out, there were minimal changes outside of the revert though.

Reply:
Thank you!